sync/class.jetpack-sync-module-themes.php

```diff
@@ -414,9 +431,34 @@ private function get_theme_support_info() {
 		}
 
 		$theme = wp_get_theme();
-		$theme_support['name'] = $theme->name;
-		$theme_support['version'] =  $theme->version;
+		$theme_support['name'] = $theme->get('Name');
+		$theme_support['version'] =  $theme->get('Version');
+		$theme_support['slug'] = $theme->get_stylesheet();
+		$theme_support['uri'] = $theme->get('ThemeURI');
+
 
 		return $theme_support;
 	}
+
+	private function get_delete_theme_call() {
```

Comment:
should we name this `is_theme_delete` and return a `bool` so both methods are similar?